src/main/webapp/app/views/admin/info/edit/input-strict_contact.html

```diff
@@ -0,0 +1,29 @@
+<div class="btn-group btn-group-justified submission-input-select" uib-dropdown keyboard-nav role="group">
+    <button type="button" class="btn dropdown-toggle dropdown-toggle-button" uib-dropdown-toggle>
+      <span class="caret"></span>
+    </button>
+    <button ng-class="{selected: fieldValue.value}" type="button" class="btn btn-default dropdown-text" disabled>
+      <span class="text-left dropdown-text-label">{{field.value || 'Choose ' + fieldProfile.gloss}}</span>
+    </button>
+    <ul class="dropdown-menu" uib-dropdown-menu role="menu">
+      <li class="select-option"
+        role="menuitem"
+        ng-click="field.value = word.name; fieldProfileForm.$dirty = true; saveWithCV(field, word);" 
```

Comment:
This might be better served with the ```field.value = ...``` inside of a single function.
(and add the appropriate tests.)

Reply:
See above comment.